indico/modules/rb/client/js/components/SidebarMenu.jsx

```diff
@@ -5,13 +5,17 @@
 // modify it under the terms of the MIT License; see the
 // LICENSE file for more details.
 
-import React from 'react';
+import fetchConfig from 'indico-url:rb.config';
```

Comment:
`configURL`, but the config is in the redux state so you do not need to fetch it in here